### PR TITLE
Update with latest German translations.

### PR DESCRIPTION
**Pull request by wbnns**
This PR updates Bitcoin.org with the latest German translations.

### CODE REVIEW

**Comment by wilbns** [comment]
@saivann Any idea what's up on the Travis error on this? Other than that, should be good to go!

File: _translations/de.yml [hunk 1]
```diff
@@ -226,7 +236,7 @@ de:
     devcommunitiesintro: "Die folgenden Chatrooms und Websites bieten DIskussionen über die Bitcoin Entwicklung. Lesen Sie bitte die Verhaltensregeln bevor Sie sich beteiligen."
     ircjoin: "<a href=\"https://webchat.freenode.net/?channels=bitcoin-dev\">IRC Channel #bitcoin-dev</a> auf freenode."
     stackexchange: "<a href=\"https://bitcoin.stackexchange.com/\">Bitcoin StackExchange</a>"
-    bitcointalkdev: "<a href=\"https://bitcointalk.org/index.php?board=6.0\">BitcoinTalk Development &amp; Technical Discussion Forum</a>"
+    bitcointalkdev: "<a href=\"https://bitcointalk.org/index.php?board=6.0\">BitcoinTalk Development & Technical Discussion Forum</a>"
```
Comment (harding) [inline _translations/de.yml]: I'm pretty sure this needs to be `&amp;` to fix the travis error.

File: _translations/de.yml [hunk 2]
```diff
@@ -605,7 +615,7 @@ de:
     spread: "Weitersagen"
     spreadtxt: "Sprechen Sie mit interessierten Menschen über Bitcoin. Schreiben Sie in Ihrem Blog darüber. Sagen Sie Ihrem Lieblingsgeschäft, dass Sie gerne mit Bitcoin bezahlen würden. Helfen Sie <a href=\"http://usebitcoins.info/\">Händlerverzeichnisse</a> aktuell zu halten. Oder seien Sie kreativ und gestalten Sie sich ein schönes Bitcoin-T-Shirt."
     wiki: "Dokumentation"
-    wikitxt: "<a href=\"https://github.com/bitcoin-dot-org/bitcoin.org#how-to-participate\">Bitcoin.org</a> und das <a href=\"https://de.bitcoin.it/\">Bitcoin Wiki</a>bieten viele nützliche und detaillierte Informationen, und wir verbesseren die Inhalte laufend - genau wie Wikipedia. Sie können helfen, das Wiki akkurat und aktuell zu halten."
+    wikitxt: "<a href=\"https://github.com/bitcoin/bitcoin.org#how-to-participate\">Bitcoin.org</a> und das <a href=\"https://de.bitcoin.it/\">Bitcoin Wiki</a>bieten viele nützliche und detaillierte Informationen, und wir verbesseren die Inhalte laufend - genau wie Wikipedia. Sie können helfen, das Wiki akkurat und aktuell zu halten."
```
Comment (harding) [inline _translations/de.yml]: This should be changed to bitcoin-dot-org

File: _translations/de.yml [hunk 3]
```diff
@@ -678,14 +687,14 @@ de:
...
     menu-press: "Presse"
     menu-privacy: "Privatsphäre"
     menu-resources: Ressourcen
-    menu-stats: "Statistiken"
-    menu-participate: Mitmachen
+    menu-support-bitcoin: "Bitcoin unterstützen"
     menu-vocabulary: Glossar
     menu-you-need-to-know: "Das sollten Sie wissen"
-    banner-translate: "Übersetzungen für diese Sprache sind veraltet. <a href=\"https://github.com/bitcoin-dot-org/bitcoin.org#translation\">Klicken Sie hier um bei der Übersetzung von bitcoin.org in ihre Sprache zu helfen</a>."
+    banner-translate: "Übersetzungen für diese Sprache sind veraltet. <a href=\"https://github.com/bitcoin/bitcoin.org#translation\">Klicken Sie hier um bei der Übersetzung von bitcoin.org in ihre Sprache zu helfen</a>."
```
Comment (harding) [inline _translations/de.yml]: This should be changed to bitcoin-dot-org

**Comment by saivann** [comment]
@wilbns Just to confirm, did you re-translate texts back to English, compared them with the original English strings, reported any potential discrepancies to the translators and updated translations if necessary?

**Comment by wilbns** [comment]
@saivann Hey there, yes, I checked the diffs on GT and did not see any significant discrepancies.

**Comment by wilbns** [comment]
@harding Thanks for catching the problem on bitcoin-dot-org vs bitcoin.org (thought it redirected). Lastly, the ampersand! :+1:

File: _translations/de.yml (hunk 1 above)
Comment (wilbns) [inline _translations/de.yml]: Fixed.

File: _translations/de.yml (hunk 2 above)
Comment (wilbns) [inline _translations/de.yml]: Updated.

**Comment by wilbns** [comment]
I'm going to update this on Transifex so everything is in sync and will push a new commit w/ the above changes.

File: _translations/de.yml (hunk 1 above)
Comment (harding) [inline _translations/de.yml]: Not fixed; I think this is the cause of the continuing Travis error.

**Pull request closed without merging**